src/Components/common/validateFormTestimonials.js

```diff
@@ -0,0 +1,18 @@
+export const validateFormTestimonials = (values) => {
+    let errors={}
+
+    if(!values.image) errors.image='You need to add a image';
+
+    if(!values.name) errors.name='You need to enter a name';
+    if(values.name.length<4) errors.name='The name must have more than 4 letters';
```

Comment:
this validation should be a separated function in string or list common file.
```
const isSmallerThan = (list, maxSize) => list.length < maxSize;
```
then you use it here
```
...
if(isSmallerThan(values.name, 4) .....
```